Examples/Examples.playground/Pages/07MapKitExtensions.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,11 @@
+//: [Table of Contents](00ToC)
```

Comment:
Also for the files for the modules we don't have examples yet, we can remove and add the files when we have some :))